starters/servers/cloudflare/rollup.config.server.js

```diff
@@ -0,0 +1,30 @@
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import { qwikRollup } from '@builder.io/qwik/optimizer';
+import jsonPlugin from '@rollup/plugin-json';
+import commonjs from '@rollup/plugin-commonjs';
+
+export default async function () {
+  return {
+    input: {
+      index: 'src/index.cloudflare.tsx',
+    },
+    inlineDynamicImports: true,
+    plugins: [
+      nodeResolve(),
+      jsonPlugin(),
+      qwikRollup({
+        entryStrategy: {
+          type: 'single',
+        },
+      }),
+      commonjs(),
+    ],
+    output: [
+      {
+        intro: 'const global = globalThis;',
```

Comment:
`const global = globalThis;` can get removed from here now right?

Reply:
weird... I thought it did!